### ISSUE TITLE
new home page

### ISSUE THREAD

**Issue by kueda** [inaturalist/inaturalist#691]
For design discussion and mocks see https://github.com/inaturalist/design/issues/35. This is mostly just an implementation todo list.

Assets: https://www.dropbox.com/home/iNaturalist/Design/Homepage
Icons from http://ionicons.com: ion-ios-location-outline
Current mock: https://cloud.githubusercontent.com/assets/11527994/9918054/9a21d066-5c79-11e5-878c-1d13ac00f3bb.jpg

- [x] bootstrap3: https://github.com/inaturalist/inaturalist/issues/674 (mostly done, just not deployed, but done enough to work from)
- [x] support custom font (Whitney)
- [x] hero
- [x] how
- [x] science
- [x] value props
- [x] people
- [x] call to action
- [x] new font for icons (using http://app.fontastic.me, see lastpass for creds)
- [x] include assets
- [x] user carousel
- [x] finalize testimonials and user images
- [x] figure out how to store data like featured observation, home-page-only assets, etc. Should this even be in the repo or a custom wiki page?
- [x] layout of stats slide in hero carousel
- [x] figure out API to get stats

**Comment by joellebel** [comment]
Just posting links to the icons, their licensing info and whether or not they were modified:

* girl (modified): https://thenounproject.com/term/girl/69716/ 
* spy (modified): https://thenounproject.com/term/spy/69739/
* geek (modified): https://thenounproject.com/term/geek/69709/
* butterfly (modified): https://thenounproject.com/term/butterfly/138812/
* moth (modified): https://thenounproject.com/term/moth/17111/
* cloud video (modified): https://thenounproject.com/term/cloud-video/68234/
* group (modified): https://thenounproject.com/term/group/176403
* owl (modified): https://thenounproject.com/term/owl/123996
* bee (modified): https://thenounproject.com/term/bee/182489
* honeycomb (modified): https://thenounproject.com/term/honeycomb/5568
* cowboy hat (modified): https://thenounproject.com/term/cowboy-hat/156402
* glasses: https://thenounproject.com/term/glasses/30742
* megaphone (modified): https://thenounproject.com/search/?q=megaphone&i=169426

**Comment by joellebel** [comment]
All assets have been uploaded here:

https://www.dropbox.com/home/iNaturalist/Design/Homepage

For the How It Works illustrations I created SVGs which I've tested on the latest versions of safari, chrome, and FF. I can always create bitmap versions, if needed.

**Comment by kueda** [comment]
@pleary, one part of this is displaying three stats:

1. total number of observations
1. total number of observed taxa
1. total number of registered, unsuspended users

I can get 1 from http://www.inaturalist.org/observations.json, but I'm wondering, is this something we can get from your work on opening up `stats_adequately_scoped`? Or are there other existing endpoints I can use? If not I'll just make one. We could also change 3 to total number of observers, or maybe total number of observers and identifiers, if though I suspect those might actually be harder.

**Comment by pleary** [comment]
I haven't changed anything yet, but we could in theory just remove most or all of the stats_adequately_scoped? method and you could hit the existing observations/taxon_stats and user_stats endpoints which should give you number of observed taxa and number of observers respectively. There is nothing yet that just counts users, as far as I know, aside from admin/stats.

The fastest way would be to get both from a single ES query, which would do a lot less and be a lot faster than calling taxon_stats and user_stats which query for other things:

```
Observation.elastic_search(size: 0,
  aggregate: {
    distinct_taxa: { cardinality: { field: "taxon.id", precision_threshold: 10000 } },
    distinct_observers: { cardinality: { field: "user.id", precision_threshold: 10000 } }
  }).response.aggregations
```

That would give you # observed taxa and observers. We don't have the data in ES for "total number of registered, unsuspended users", so you'd need to query the DB for that: `User.where("confirmed_at IS NOT NULL AND suspended_at IS NULL").count`

**Comment by joellebel** [comment]
Looking so good... I'm working on the photos now.

### Punch List

- [ ] Slideshow: The shell should be second with the correspond blue box and the bird should be last with it's corresponding yellow box

- [ ] Remove usernames because it's too wordy

- [ ] The ob photos themselves should link to its ob

- [ ]  Works on All Your Devices section needs a 30px trimming on the bottom (See blue guide)

![image](https://cloud.githubusercontent.com/assets/11527994/10146852/c2726bd8-65df-11e5-9a2c-ce0dc12de3ac.png)

- [ ] Works on all devices image: Shouldn't go to the left beyond this point. See screenshot...
![image](https://cloud.githubusercontent.com/assets/11527994/10147669/31456b7e-65e4-11e5-9270-2389c1605cae.png)


- [ ] Works on all devices text: it shouldn't go beyond this point. See screenshot...

![image](https://cloud.githubusercontent.com/assets/11527994/10147635/ff4cb226-65e3-11e5-8a51-4a42d7e0ffe3.png)


- [ ] Works on all devices blurb: We talked about adding "...or wifi." or something like that to make the blurb fit the space better. Plus, I think it's good we mention wifi.

- [ ] Who Uses iNaturalist?: Since we're going with longer quotes I would make the slide pause for 6 secs even though folks have control over the slider. Lots of times they miss it

- [ ]  Who Uses iNaturalist?: Title and location need the following styles and arrow icon...

![image](https://cloud.githubusercontent.com/assets/11527994/10147770/e1d5c880-65e4-11e5-80ac-ef11d97cb554.png)


- [ ] Slide Indicators: Need the following styles...

![image](https://cloud.githubusercontent.com/assets/11527994/10147783/f910016e-65e4-11e5-8413-bd02d14c88e1.png)

- [ ] This button should be 60px high. Right now it's only 55px. After making it larger, please ensure it is vertically middle-aligned within the button
![image](https://cloud.githubusercontent.com/assets/11527994/10147897/85250f1e-65e5-11e5-9c86-7b082cc98312.png)

**Comment by joellebel** [comment]
The profile photos ready and here on dropbox:
Design / Homepage / Quotes

**Comment by joellebel** [comment]
- [ ] Link in "Contribute..." needs to be white with underline

- [ ] Join button needs to go back to Whitney Book 

- [ ] Value Prop Spacing: Can we make the icons a little closer to their blurbs and more space in between the two columns so it's clear which icons are referring to which text.

- [ ] The quote person's name needs 8px of padding below it. 

- [ ] There needs to be about 50% more space between the "...reception or wifi." and the play store buttons. 

...and then I think that's a wrap!

**Comment by carrieseltzer** [comment]
Wow, I finally took at look at the few images I could see on github and it looks AWESOME! I am so excited to see this for real! Will it go on gorilla first or are you going to surprise us?

**Comment by joellebel** [comment]
@carrieseltzer thanks! I am very excited to see it launch as well :)

**Comment by naturalistcharlie** [comment]
Yeah, excited to see this... of course :)

On Tue, Sep 29, 2015 at 11:32 PM, joellebel <notifications@github.com>
wrote:

> @carrieseltzer <https://github.com/carrieseltzer> thanks! I am very
> excited to see it launch as well :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inaturalist/inaturalist/issues/691#issuecomment-144272883>
> .
>



-- 
============================
Charlie Hohn
Montpelier, Vermont

**Comment by carrieseltzer** [comment]
After seeing it on gorilla (thanks Ken-ichi!), a few comments/suggestions:

-Add commas to the stats of observations, users, and species. It's easier to read numbers that way.
-Hero images in the top carousel don't link out to functional links (search mole) but I'm guessing that's just because it's gorilla and it will work on the real thing.
-Make link to GBIF open in a new tab?
-User slide carousel moves a little fast--maybe add 1 more sec?
-Danielle should write something in her profile.

Slightly OT: The heat map of all observations (http://www.inaturalist.org/observations/map) is really not easily accessible, but it's a super cool thing to explore. I'd love to see it featured more prominently somewhere on the site.

Really great work! What's the expected release date?

**Comment by kueda** [comment]
> Add commas to the stats of observations, users, and species. It's easier to read numbers that way.

I see commas. Screenshot and browser?

> Hero images in the top carousel don't link out to functional links (search mole) but I'm guessing that's 
just because it's gorilla and it will work on the real thing.

Gorilla's DB is just out of date, will update.

> Really great work! What's the expected release date?

Soonish.

**Comment by carrieseltzer** [comment]
You're right, I see them in Chrome and Firefox but not in Safari Version 8.0.7 (10600.7.12). Screenshot below.
![screen shot 2015-09-30 at 1 30 58 pm](https://cloud.githubusercontent.com/assets/7501183/10200884/02f66356-6777-11e5-8bea-407e2eff683a.png)

**Comment by carrieseltzer** [comment]
Commas also missing from mobile safari. Is there a mobile view plan for the new homepage?







—

**Comment by kueda** [comment]
Ok, try gorilla again. I think I dealt with the Safari issues.

**Comment by carrieseltzer** [comment]
Yep! Commas are there on mobile and non-mobile safari.

**Comment by joellebel** [comment]
- [ ] Please capitalize "Uses" in "Who uses iNat.." for consistency
- [ ] All body copy should be 500 weight. Light will hardly be used on the site so maybe 500 weight should be "normal"? Up to you. (I just dont want light to sneak it's way back in there ;).
- [ ] Please change color of headings to #1A1A1A
- [ ] Slideshow: Please make the icons and text blurbs in the boxes clickable to their corresponding Meet, Explore, etc.  pages.
